### PR TITLE
N8N-2961 New expression variables

### PR DESCRIPTION
**Pull request by michael-radency**

### CODE REVIEW

File: packages/workflow/src/WorkflowDataProxy.ts [hunk 1]
```diff
@@ -446,7 +449,130 @@ export class WorkflowDataProxy {
...
+								return (branchIndex?: number, runIndex?: number) => {
+									const executionData = getNodeOutput(nodeName, branchIndex, runIndex);
+									if (executionData[0]) return executionData[0];
+									return undefined;
+								};
+							}
+							if (property === 'last') {
+								return (branchIndex?: number, runIndex?: number) => {
+									const executionData = getNodeOutput(nodeName, branchIndex, runIndex);
+									if (!executionData.length) return [];
```
Comment (krynble) [inline packages/workflow/src/WorkflowDataProxy.ts:503]: Just to be consistent, we should return `undefined` here

**Review by krynble — CHANGES_REQUESTED** [review]
Hey @michael-radency amazing job! I just added a tiny change for the sake of consistency.

**Review by krynble — CHANGES_REQUESTED** [review]

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.167.0`

**Pull request merged**